library/src/main/java/com/getstream/sdk/chat/adapter/AttachmentViewHolder.java

```diff
@@ -69,7 +69,7 @@ public void bind(@NonNull Context context,
         this.context = context;
         this.messageListItem = messageListItem;
         this.message = message;
-        this.attachment = attachment;
+        this.attachment = attachmentItem.getAttachment();
```

Comment:
Decided to not change the type of the private fields inside the ViewHolder classes, for now. This can be changed later so that the entire item is stored if needed, but that's all private implementation details, and it doesn't break the public API.